tos/tests/test_models.py

```diff
@@ -1,26 +1,26 @@
+from django.contrib.auth import get_user_model
```

Comment:
Thanks for the feedback.  I'll work on a patch that supports 1.4.


On 2015-08-05 14:30, Daniel Greenfeld wrote:
> In tos/tests/test_models.py [1]:
> 
>> @@ -1,26 +1,26 @@
>> +from django.contrib.auth import get_user_model
> 
> Since Django 1.4 is still on the supported list at least for a little
> while longer, this project should continue to support.
> 
> --
> Reply to this email directly or view it on GitHub [2].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/revsys/django-tos/pull/9#discussion_r36356702
> [2] https://github.com/revsys/django-tos/pull/9/files#r36356702